lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -426,53 +362,48 @@ void onContainerRequest(final AMRMClient.ContainerRequest... containerRequests)
    *
    * @param container newly allocated
    */
-  private void handleNewContainer(final Container container, final boolean isRecoveredContainer) {
+  private void handleNewContainer(final Container container) {
```

Comment:
Because this portion is just for new containers. If it's a recovered container, all the logic should be handled in `DriverRestartManager.informAboutEvaluatorsAlive`.